lib/app_manifest/addon.rb

```diff
@@ -0,0 +1,10 @@
+module AppManifest
+  class Addon
+    include Virtus.model
+    include Serializer
+
+    attribute :plan,    String
+    attribute :as,      String
+    attribute :options, Hash
```

Comment:
`Hash[String => String]` or however virtus spells it? The options hash does have a single "correct" type, at least in principle.